src/content/doc-surrealql/statements/define/access/bearer.mdx

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_position: 10
+sidebar_label: BEARER 
+title: DEFINE ACCESS ... TYPE BEARER statement | SurrealQL
+description: A bearer access method allows accessing SurrealDB using a bearer key.
+---
+
+import Since from '@components/shared/Since.astro'
+
+# `DEFINE ACCESS ... TYPE BEARER`
+
+<Since v="v2.2.0" />
+
+> [!CAUTION]
+> Currently, the `BEARER` access method is an experimental feature intended to be used for validating its suitability and security. As such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this and other experimental features related to bearer access, set the `SURREAL_EXPERIMENTAL_BEARER_ACCESS` environment variable to `true`.
+
+A bearer access method allows generating bearer grants with an associated key that can be used to access SurrealDB as a specific [system user](/docs/surrealdb/security/authentication#system-users) or [record user](/docs/surrealdb/security/authentication#record-users). Bearer grants allow other systems and software to authenticate with SurrealDB using a secure and unique credential that can be [audited](/docs/surrealql/statements/access/#show) and [revoked](/docs/surrealql/statements/access/#revoke) at any time.
+
+Allowing access to SurrealDB using a bearer access method requires creating grants associated with that access method. This can be done using the [`GRANT`](/docs/surrealql/statements/access/#grant) clause of the [`ACCESS`](/docs/surrealql/statements/access) statement.
+
+After creating a grant for a subject (i.e. a [system user](/docs/surrealdb/security/authentication#system-users) or a [record user](/docs/surrealdb/security/authentication#record-users)) with a bearer access method, a bearer key will be returned. This bearer key can be used to sign in as the subject of the grant without using its password or any other credentials. As with other credentials in SurrealDB, signin in with a bearer key will return a JWT, which can be used to perform authenticated operations or establish a persistent [authenticated session](/docs/surrealdb/security/authentication/#sessions). This makes bearer keys most suitable for automations and other service-to-service authentication use cases that require interacting with SurrealDB in an authenticated context by providing stronger security guarantees than passwords and removing the complexity of having to work with JWT directly.
```

Comment:
```suggestion
After creating a grant for a subject (i.e. a [system user](/docs/surrealdb/security/authentication#system-users) or a [record user](/docs/surrealdb/security/authentication#record-users)) with a bearer access method, a bearer key will be returned. This bearer key can be used to sign in as the subject of the grant without using its password or any other credentials. As with other credentials in SurrealDB, signin with a bearer key will return a JWT, which can be used to perform authenticated operations or establish a persistent [authenticated session](/docs/surrealdb/security/authentication/#sessions). This makes bearer keys most suitable for automations and other service-to-service authentication use cases that require interacting with SurrealDB in an authenticated context by providing stronger security guarantees than passwords and removing the complexity of having to work with JWT directly.
```

Reply:
I fixed the typo in [ca77cb3](https://github.com/surrealdb/docs.surrealdb.com/pull/1078/commits/ca77cb319723f39408001bfbb20ce934c93aaa6b).